cmd/sonictool/app/sign_genesis.go

```diff
@@ -57,18 +60,20 @@ func signGenesis(ctx *cli.Context) error {
 	return nil
 }
 
-func getGenesisHeaderHashes(genesisFile string) (ogenesis.Header, ogenesis.Hashes, error) {
+func getGenesisHeaderHashes(genesisFile string) (header ogenesis.Header, genesisHashes ogenesis.Hashes, err error) {
```

Comment:
nit: Using named return values makes functions more complicated to read -- and this one in particular. Consider rewriting this into a version not using named return values.

Reply:
added return values to each return.